### ISSUE TITLE
Set up OpenXR display

### ISSUE THREAD

**Issue by marceline-cramer** [mondradiko/mdo-core#21]
Dependent on #19 

There are several steps that need to be figured out as we go along:
- link OpenXR loader library
- set up OpenXR system and session
- initialize in `cli/main.c`
- initialize `camera` and `viewport`s
- wait, begin, submit, and end frames

**Comment by MilesLitteral** [comment]
Camera is initialized in cli/main.c?

**Comment by marceline-cramer** [comment]
No, `openxr_display` is initialized, and the display creates the camera, which is retrieved by `cli/main.c` to get the initial render pass to feed to the renderer.

This is temporary, in the future, the renderer will dynamically consume displays at frame time instead of initialization time.
